pkg/client/introduce/client_test.go

```diff
@@ -53,106 +56,250 @@ func TestNew(t *testing.T) {
 }
 
 func TestClient_SendProposal(t *testing.T) {
+	const UUID = "382a7cf8-2c57-4f2f-9359-8ac45b7b4b1f"
+
 	svc, err := introduce.New(&protocol.MockProvider{})
 	require.NoError(t, err)
 	require.NotNil(t, svc)
 
 	ctrl := gomock.NewController(t)
 	defer ctrl.Finish()
 
-	provider := mocks.NewMockProvider(ctrl)
 	DIDComm := serviceMocks.NewMockDIDComm(ctrl)
 	DIDComm.EXPECT().HandleOutbound(gomock.Any(), gomock.Any()).Return(nil)
-	provider.EXPECT().Service(introduce.Introduce).Return(DIDComm, nil)
+
+	opts := options{
+		Dests: []*service.Destination{
+			{ServiceEndpoint: "service/endpoint1"},
+			{ServiceEndpoint: "service/endpoint2"},
+		},
+	}
+
+	store := storageMocks.NewMockStore(ctrl)
+	store.EXPECT().Put("client_options_"+UUID, toBytes(t, opts)).Return(nil)
```

Comment:
any reason for not using `optionsPrefix` instead of the string literal `"client_options_"` here?